src/overlays/actors/ovl_En_Horse_Link_Child/z_en_horse_link_child.c

```diff
@@ -39,9 +41,21 @@ ActorInit En_Horse_Link_Child_InitVars = {
     (ActorFunc)EnHorseLinkChild_Draw,
 };
 
-AnimationHeader* D_808DFEC0[] = {
-    &object_horse_link_child_Anim_006D44, &object_horse_link_child_Anim_007468, &object_horse_link_child_Anim_007D50,
-    &object_horse_link_child_Anim_0043AC, &object_horse_link_child_Anim_002F98,
+typedef enum {
+    OOT_EPONA_ANIMATION_IDLE,   // head down + pawing
```

Comment:
```suggestion
    /* 0 */ OOT_EPONA_ANIM_IDLE,   // head down + pawing
```
Enum value comment, as well as shorten animation to `Anim`